src/shared/reducers/ui.js

```diff
@@ -14,6 +15,7 @@ import type { UIState } from '../flow-types';
 const defaultUserState = {
     hasSeenHelpBoxType1: false,
     kmTillNextLevel: 0,
+    languageCode: 'xx',
```

Comment:
should the default language code be `xx` or `en`. I'm not sure if it is even used, but xx could be confusing.

Reply:
I used `xx` here to help detect that no language has been selected by the user. I guess I could have set it to `undefined` instead. Will fix in next release, just in case I mess something up. https://github.com/mapswipe/mapswipe/issues/370